scss.py

```diff
@@ -0,0 +1,35 @@
+"""
+Single-file libsass wrapper that can compile SASS/SCSS files into CSS.
+
+Usage: `python scss.py input.scss:output.css input2.scss:output2.css`
+"""
+
+import sys
+
+import sass
+
+if not len(sys.argv) > 1:
```

Comment:
this is weird. maybe it's just me but this feels a bit like doing `if not x == 3` instead of `if x != 3`. Why not just use `if len(sys.argv) <= 1`?